packages.yml

```diff
@@ -1,6 +1,9 @@
 packages:
   - package: fishtown-analytics/dbt_utils
```

Comment:
this package reference is not required, as the source package references dbt_utils already.